webrtc/interfaces.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>IDL check of RTCPeerConnection</title>
+    <link rel="author" title="Soares Chen" href="mailto:soares.chen@gmail.com"/>
+    <link rel="help" href="http://w3c.github.io/webrtc-pc/">
+  </head>
+  <body>
+    <h1 class="instructions">Description</h1>
+    <p class="instructions">This test verifies the conformance of WebIDLs defined in WebRTC</p>
+    <div id="log"></div>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+    <script>
+     (() => {
+       'use strict'
+
+       function initRTCPeerConnection(idlArray) {
```

Comment:
This is quite a bit more verbose than necessary for most of these objects that are trivial to create. Like https://github.com/ayg/web-platform-tests/blob/master/XMLHttpRequest/interfaces.html, you could do something like this for most of it:
```JavaScript
  idlArray.add_objects({
    RTCPeerConnection: ['new RTCPeerConnection'],
    RTCIceCandidate: ['new RTCIceCandidate'],
    ...
  });
```

Try to minimize the amount of code. You probably noticed that `new RTCIceCandidate({})` throws an exception in both Chrome and Firefox because the `candidate` member is missing. Be sure to not work around such issues, but instead file issues on the spec or implementations, depending on which you think is wrong.